SECURITY.md

```diff
@@ -0,0 +1,39 @@
+# Security Policy
+
+## Reporting a Vulnerability
+
+If you believe you have found a security vulnerability in `xclim`, we encourage you to let us know right away. We take all security vulnerabilities seriously and appreciate your efforts to responsibly disclose them.
```

Comment:
Here you mention xclim, but this applies to all repos, right ?

Reply:
Good catch, will update!